tkupchyn/homework_08/three_chefs.py

```diff
@@ -0,0 +1,43 @@
+# Details:  https://py.checkio.org/mission/3-chefs/share/33d5c11b5ff002a1645a6ee3b93ce00a/
+
+
+class AbstractCook:
+
+    def __init__(self):
+
+        self.total_food = 0
+        self.total_drinks = 0
+        self.food_name = 'Food'
+        self.drink_name = 'Drinks'
+
+    def total(self):
+        return (f'{self.food_name}: {self.total_food},'
+                f' {self.drink_name}: {self.total_drinks},'
+                f' Total: {self.total_food + self.total_drinks}')
+
+    def add_food(self, food_amount, food_price):
+        self.total_food += food_amount * food_price
+
+    def add_drink(self, drink_amount, drink_price):
+        self.total_drinks += drink_amount * drink_price
```

Comment:
your solution is ok, but it does not scale well if to add other dishes
think in sepparate pr how could you solve this problem if you add bevarage, snack, desert etc.